lib/components/ConflictPicker.vue

```diff
@@ -105,11 +107,18 @@ export default Vue.extend({
 			default: '',
 		},
 
+		/** All the existing files in the current directory */
+		content: {
+			type: Array as () => Node[],
```

Comment:
```suggestion
			type: Array as PropType<Node[]>,
```
Wrapping with PropType generic seems to be the Vue + TypeScript way